packages/components/src/components/ContentfulContainer/ContentfulContainer.tsx

```diff
@@ -6,45 +6,32 @@ import { ContentfulContainerAsHyperlink } from './ContentfulContainerAsHyperlink
 import type { ContentfulContainerAsHyperlinkProps } from './ContentfulContainerAsHyperlink';
 import { combineClasses } from '../../utils/combineClasses';
 
-export const ContentfulContainer: React.FC<ContentfulContainerAsHyperlinkProps> = (
-  sectionProps
-) => {
-  const { className, editorMode, children } = sectionProps;
+export const ContentfulContainer: React.FC<ContentfulContainerAsHyperlinkProps> = (props) => {
+  const { className, editorMode, children, cfHyperlink } = props;
```

Comment:
slight nitpick, but I generally like to destructure the props in the function params. Its slightly cleaner (imo) and you don't have an additional props param that might cause some confusion later in scope of the component.

Reply:
The way the `ContentfulContainerAsHyperlinkProps` is defined conditionally based on `editorMode` makes it difficult to move this destructuring to the function params.  I think I'll leave this as-is